trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/empty.py

```diff
@@ -82,6 +83,12 @@ def empty(vehicle):
           t_c - thickness-to-chord ratio of the vertical tail                    [dimensionless]
           sweep - sweep angle of the vertical tail                               [radians]
           t_tail - factor to determine if aircraft has a t-tail, "yes"           [dimensionless]
+          
+      settings.weight_reduction_factors.
+          main_wing                                                              [dimensionless] (.1 is a 10% weight reduction)
+          empennage                                                              [dimensionless] (.1 is a 10% weight reduction)
```

Comment:
should dimensionless be [-]?

Reply:
That's what I usually use but we don't have a standard. I matched it to the rest of the file.